src/fzx/tui/term_app.cpp

```diff
@@ -143,4 +169,16 @@ void TermApp::quit()
   mQuit = true;
 }
 
+void TermApp::printSelection()
+{
+  for (const auto& thing : mSelection) {
+    std::cout << thing << std::endl;
+  }
+}
+
+std::string_view TermApp::currentItem() const {
+  return mFzx.getResult(mCurpos).mLine;
```

Comment:
This can fail when there are no items matched, so maybe just return null string_view then?
```suggestion
  if (mCurpos < mFzx.resultsSize())
    return mFzx.getResult(mCurpos).mLine;
  return {};
```